capsules/extra/src/servo.rs

```diff
@@ -19,7 +19,7 @@
 //! capsules_extra::sg90::Sg90<
 //!    'static,
 //!    capsules_core::virtualizers::virtual_pwm::PwmPinUser<'static, rp2040::pwm::Pwm>,
-//! >,
+//! \>,
```

Comment:
I think that instead of this, we should make this snippet into an actual code block. You might want to annotate it so that it's not interpreted as a doctest though.

Reply:
I have attempted to wrap this code block in a backtick ignore block as per => https://users.rust-lang.org/t/disabling-rustdoc-tests-for-module/50110/9